packages/releaser/src/ReleaseWorker/CheckPackagesTravisBuildsReleaseWorker.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker;
+
+use PharIo\Version\Version;
+use Safe\simplexml_load_string;
+use Shopsys\Releaser\Guzzle\ApiCaller;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symplify\MonorepoBuilder\Release\Contract\ReleaseWorker\ReleaseWorkerInterface;
+
+/**
+ * @see https://stackoverflow.com/questions/34277366/how-to-list-all-builds-of-a-given-project-through-travis-api
+ * @see http://docs.guzzlephp.org/en/stable/quickstart.html#concurrent-requests
+ */
+final class CheckPackagesTravisBuildsReleaseWorker implements ReleaseWorkerInterface
+{
+    /**
+     * @var string[]
+     */
+    private $travisPackages = [];
+
+    /**
+     * @var string[]
+     */
+    private $passesPackages = [];
+
+    /**
+     * @var string[]
+     */
+    private $failedPackages = [];
+
+    /**
+     * @var \Symfony\Component\Console\Style\SymfonyStyle
+     */
+    private $symfonyStyle;
+
+    /**
+     * @var string
+     */
+    private const SUCCESS_STATUS = 'Success';
+
+    /**
+     * @var \Shopsys\Releaser\Guzzle\ApiCaller
+     */
+    private $apiCaller;
+
+    /**
+     * @param \Symfony\Component\Console\Style\SymfonyStyle $symfonyStyle
+     * @param string[] $travisPackages
+     * @param \Shopsys\Releaser\Guzzle\ApiCaller $apiCaller
+     */
+    public function __construct(SymfonyStyle $symfonyStyle, array $travisPackages, ApiCaller $apiCaller)
+    {
+        $this->travisPackages = $travisPackages;
+        $this->symfonyStyle = $symfonyStyle;
+        $this->apiCaller = $apiCaller;
+    }
+
+    /**
+     * @return string
+     */
+    public function getDescription(): string
+    {
+        return 'Check Travis build status for all packages';
+    }
+
+    /**
+     * Higher first
+     * @return int
+     */
+    public function getPriority(): int
+    {
+        return 1000;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     */
+    public function work(Version $version): void
+    {
+        // @todo for fast development
+        return;
```

Comment:
could this worker also force execute builds of all packages? 
WHY: we have packages that have dependencies for example on shopsys/framework, but when shopsys/framework changes and have new commit in master branch on packagist these packages are not executed so we don't know whether these packages will work with the current version of (composer) of framework or whatever.

It is very hard to check whether all master branches are OK because defaultly in travis there can be green branches that are not from master branch and we need to get into the master version and then rebuild the whole set for php 7.1 and 7.2 and also for http-smoke-tests 5.6 .......